rts/motoko-rts/src/gc.rs

```diff
@@ -22,5 +22,5 @@ unsafe fn should_do_gc(max_live: crate::types::Bytes<u64>) -> bool {
         (u64::from(LAST_HP) + max_live.0) / 2,
     );
 
-    u64::from(getHP()) >= heap_limit
+    u64::from(getHP() + 1) >= heap_limit
```

Comment:
Using `unskew()` or `skew()` from `types.rs` could make this more self-explanatory. Or alternatively, probably even better, `getHP()` could return `Value`.

Reply:
Good point, will do. (But not `Value`.)

Reply:
done in 116d73596